plugin/ZoomableThumbnail.qml

```diff
@@ -24,19 +24,17 @@ SilicaFlickable {
     id: base;
 
     property real maxHeight: height;
+    property alias content: thumb.data;
```

Comment:
Nevermind about semi-colons. File seems to be full of them. Worth adding a cleanup task to github repo.